Lib9c/Action/AccountStateDeltaExtensions.cs

```diff
@@ -297,5 +298,39 @@ public static RedeemCodeState GetRedeemCodeState(this IAccountStateDelta states)
                 throw;
             }
         }
+
+        public static T GetSheet<T>(this IAccountStateDelta states) where T : ISheet, new()
+        {
+            var address = Addresses.GetSheetAddress<T>();
+            var value = states.GetState(address);
+            if (value is null)
+            {
+                Log.Warning($"{nameof(T)} is null ({0})", address.ToHex());
+                throw new NullReferenceException();
```

Comment:
`NullReferenceException` may not fit in this case. (because we didn't access `value`). we may need to define dedicated class like `SheetStateMissingException`.

Reply:
Can i use [FailedLoadStateException](https://github.com/planetarium/lib9c/blob/2e645be18a4e2caea031c347f00777fbad5dbcc6/Lib9c/Action/FailedLoadStateException.cs#L7)?

Reply:
It seems appropriate.